src/electron/device-connect-view/components/device-connect-connected-device.tsx

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { Spinner, SpinnerSize } from 'office-ui-fabric-react/lib/Spinner';
+import * as React from 'react';
+
+export interface DeviceConnectConnectedDeviceProps {
+    isConnecting: boolean;
+    connectedDevice?: string;
+}
+
+export class DeviceConnectConnectedDevice extends React.Component<DeviceConnectConnectedDeviceProps> {
```

Comment:
Can be a `NamedSFC` functional component!

Reply:
Now a `NamedSFC` functional component!